napari/experimental/_progressive_loading.py

```diff
@@ -859,6 +864,9 @@ def set_interval(self, coords):
 
             
             max_where = np.where(cumuchunks >= self._max_coord[dim])
+            # LOGGER.debug(f'max_where: {max_where}')
+            if not max_where[0].any():
+                pass
```

Comment:
This is where we fail if we attempt to `set_interval` with bad values (before the initial canvas is rendered). I suspect its an out of bounds issue, but its not caught by the "no visible data" fix (https://github.com/kephale/napari/pull/19/files#diff-a9411a21e3c761868d41b8ea7efab022cb21cdf8c75669781e21df4f0a1a628bR186-R189)